src/sc_client/models.py

```diff
@@ -224,7 +224,13 @@ def get_index(self, alias: str) -> int:
         return self.aliases[alias]
 
 
-SCsText = List[str]
+class SCs:
+    def __init__(self, text: str, output_struct: ScAddr = ScAddr(0)) -> None:
+        self.scs = text
+        self.output_struct = output_struct
```

Comment:
```
class ScTemplateValue(TypedDict):
    value: ScTemplateParamValue
    alias: str | None
```
Make _SCs_ class similar to others

Reply:
I change to dataclass cause TypedDict does not support instance and class checks